dask/array/core.py

```diff
@@ -234,37 +238,58 @@ def getem(
     lock=False,
     asarray=True,
     dtype=None,
+    inline_array=True,
 ):
     """Dask getting various chunks from an array-like
 
-    >>> getem('X', chunks=(2, 3), shape=(4, 6))  # doctest: +SKIP
+    >>> graph_from_arraylike('X', chunks=(2, 3), shape=(4, 6))  # doctest: +SKIP
     {('X', 0, 0): (getter, 'X', (slice(0, 2), slice(0, 3))),
      ('X', 1, 0): (getter, 'X', (slice(2, 4), slice(0, 3))),
      ('X', 1, 1): (getter, 'X', (slice(2, 4), slice(3, 6))),
      ('X', 0, 1): (getter, 'X', (slice(0, 2), slice(3, 6)))}
 
-    >>> getem('X', chunks=((2, 2), (3, 3)))  # doctest: +SKIP
+    >>> graph_from_arraylike('X', chunks=((2, 2), (3, 3)))  # doctest: +SKIP
     {('X', 0, 0): (getter, 'X', (slice(0, 2), slice(0, 3))),
      ('X', 1, 0): (getter, 'X', (slice(2, 4), slice(0, 3))),
      ('X', 1, 1): (getter, 'X', (slice(2, 4), slice(3, 6))),
      ('X', 0, 1): (getter, 'X', (slice(0, 2), slice(3, 6)))}
     """
     out_name = out_name or arr
```

Comment:
Yeah, in the original implementation it could be a key *or* an actual value. But here it's now *only* array-like values. I've clarified the docstring.